### PR TITLE
✨ [PRTL-652] add commit and tag to footer

### PR DESCRIPTION
**Pull request by Jephuff**

### CODE REVIEW

**Comment by Jephuff** [comment]
this + https://github.com/NCI-GDC/tungsten/pull/526 will add the commit hash

File: modules/node_modules/@ncigdc/dux/versionInfo.js [hunk 1]
```diff
@@ -1,48 +1,33 @@
 // @flow
-/* global __GDC_API__, __COMMIT_HASH__, __VERSION_TAG__, __COMMIT_LINK__ */
-
 import { fetchApi } from '@ncigdc/utils/ajax';
 import { handleActions } from 'redux-actions';
-import packageJSON from './package.json';
+import packageJSON from '../../../../package.json';
```
Comment (shanewilson) [inline modules/node_modules/@ncigdc/dux/versionInfo.js]: this won't work once deployed

Comment (shanewilson) [inline modules/node_modules/@ncigdc/dux/versionInfo.js]: can stick with just the commit - I think the best solution is to check if a commit is a tag and then use the tag and if not show the commit. I just noticed that having a tag and a commit hash is confusing to people. but that can happen later

Comment (Jephuff) [inline modules/node_modules/@ncigdc/dux/versionInfo.js]: no? why not? the path? or bundling the package.json?

Comment (shanewilson) [inline modules/node_modules/@ncigdc/dux/versionInfo.js]: package.json doesn't get bundled and browsers can't access the file system

Comment (alex-wilmer) [inline modules/node_modules/@ncigdc/dux/versionInfo.js]: Oh I thought maybe we had a json loader in our webpack pipeline. Would be convenient to have no?

Comment (alex-wilmer) [inline modules/node_modules/@ncigdc/dux/versionInfo.js]: it works like this for submission ui

Comment (Jephuff) [inline modules/node_modules/@ncigdc/dux/versionInfo.js]: Why wouldn't it be bundled if it's imported? 
Isn't deploying just using the build script in the package.json?

**Comment by Jephuff** [comment]
👍  updated

**Pull request merged**